auslib/web/admin/swagger/api.yaml

```diff
@@ -126,6 +132,33 @@ parameters:
     x-nullable: true
     required: false
 
+  productRequiredParam:
+    name: product
+    in: query
+    description: Name of the product to filter the rules/releases against.
+    type: string
+    minLength: 1
+    maxLength: 15
+    required: true
+
+  roleRequiredParam:
+    name: role
+    in: query
+    description: roles are used when signing off on Scheduled Changes
+    type: string
+    minLength: 1
+    maxLength: 50
+    required: true
+
+  channelRequiredParam:
+    name: channel
+    in: query
+    description: The update channel of the application request an update.
```

Comment:
I borrowed the definition from the [database model in docs](http://mozilla-balrog.readthedocs.io/en/latest/database.html). What should I change it to ?